actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -718,7 +795,15 @@ def method_missing(method_sym, *args, &block)
       end
     end
 
-    # Returns duplicate of object including all parameters
+    def respond_to?(name, include_all = false) # :nodoc:
```

Comment:
It can't be `respond_to_missing?` because `to_hash` is implemented and `respond_to?` would return `true` without calling `respond_to_missing?`